epub33/core/index.html

```diff
@@ -8071,12 +8067,37 @@ <h3>Navigation Document Overlays</h3>
 				</div>
 			</section>
 		</section>
-		<!-- <section id="sec-privacy">
-			<h2>Privacy</h2>
-			<div class="ednote">
-				<p>The working group will address issues related to privacy of EPUB Publications in a future draft.</p>
+		<section id="sec-accessibility">
+			<h2>Accessibility</h2>
```

Comment:
Shouldn't this section be also informative, i.e., not normative?

Reply:
Not unless we move the recommendation out of it again. It feels kind of strange to recommend accessibility in one place but talk about it another, but we could point to the recommendation where it was.

Reply:
You may be right. The only contrast that may come to the fore is that this section very much reads like the, e.g., security and privacy sections, and those are indeed informative...

The only worry I have is how would we enforce this if it is normative. But we may want to leave this open for now and see that later.

Reply:
> The only worry I have is how would we enforce this if it is normative.

Enforcing accessibility is better left to legislation, but just because you can't validate every accessibility check doesn't mean we shouldn't recommend accessibility. What havoc would it wreak on scripting if we have to test what scripts are doing?

We also have the Ace validator to show that we do, in fact, have an implementation of a checker for machine-identifiable issues.

But, looking at the RS spec, the scripting considerations are, in fact, separate from scripting support, so maybe it isn't such a bad thing. Let me try a rewrite or three.

Reply:
+1